collectors/metrics/pkg/forwarder/forwarder.go

```diff
@@ -219,9 +223,21 @@ func NewWorkerMetrics(reg *prometheus.Registry) *workerMetrics {
 			Name: "federate_filtered_samples",
 			Help: "Tracks the number of samples filtered per federation",
 		}),
-		gaugeFederateErrors: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-			Name: "federate_errors",
-			Help: "The number of times forwarding federated metrics has failed",
+		federateRequests: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "federate_requests_total",
```

Comment:
I assume there is nothing (alert, dashboard) that could break with this change of name?

Reply:
Nothing I could find in this repo at least.